std/assembly/math.ts

```diff
@@ -1,4 +1,10 @@
+/// <reference path="./util/seedrandom.d.ts" />
+
+import { ArrayBufferView } from "./arraybuffer";
```

Comment:
By doing that, we can `changetype<ArrayBufferView>` anything accessing `.dataStart`, getting rid of TS syntax highlighting issues.